eclass/latex-package.eclass

```diff
@@ -105,28 +93,28 @@ latex-package_src_doinstall() {
 
 	# This actually follows the directions for a "single-user" system
 	# at http://www.ctan.org/installationadvice/ modified for gentoo.
-	[ -z "$1" ] && latex-package_src_install all
+	[[ -z $1 ]] && latex-package_src_install all
```

Comment:
for consistency, I would always write `[[ -z ${1} ]]` (which is what you're doing yourself)

Reply:
// And I trying to always do `"${1}"`, so it will never be splitted under any circumstances :D